src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +14,20 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+
+        String[] all = text.replaceAll("[^0-9 -]", " ").split(" ");
+        List<Integer> allInt = new ArrayList<>();
+        for (String s : all) {
+            try {
```

Comment:
you don't need to use exceptions here, there is more simple way 
hing: try to use ".trim()"